packages/storybook/stories/va-table.stories.jsx

```diff
@@ -75,6 +76,180 @@ const defaultArgs = {
   'descending': false,
 };
 
+const paginationData = [
+  [
+    '04/05/2019',
+    '$3,261.10',
+    'Compensation & Pension - Recurring',
+    'Direct Deposit',
+    'BANK OF AMERICA, N.A.',
+    '**3456',
+  ],
+  [
+    '03/05/2019',
+    '$3,261.10',
+    'Compensation & Pension - Recurring',
+    'Direct Deposit',
+    'BANK OF AMERICA, N.A.',
+    '**3456',
+  ],
+  [
+    '02/05/2019',
+    '$3,261.10',
+    'Compensation & Pension - Recurring',
+    'Direct Deposit',
+    'BANK OF AMERICA, N.A.',
+    '**3456',
+  ],
+  [
+    '01/05/2019',
+    '$3,261.10',
+    'Compensation & Pension - Recurring',
+    'Direct Deposit',
+    'BANK OF AMERICA, N.A.',
+    '**3456',
+  ],
+  [
+    '12/05/2018',
+    '$3,261.10',
+    'Compensation & Pension - Recurring',
+    'Direct Deposit',
+    'BANK OF AMERICA, N.A.',
+    '**3456',
+  ],
+  [
+    '11/05/2018',
+    '$3,261.10',
+    'Compensation & Pension - Recurring',
+    'Direct Deposit',
+    'BANK OF AMERICA, N.A.',
+    '**3456',
+  ],
+  [
+    '10/05/2018',
+    '$3,261.10',
+    'Compensation & Pension - Recurring',
+    'Direct Deposit',
+    'BANK OF AMERICA, N.A.',
+    '**3456',
+  ],
+  [
+    '09/05/2018',
+    '$3,261.10',
+    'Compensation & Pension - Recurring',
+    'Direct Deposit',
+    'BANK OF AMERICA, N.A.',
+    '**3456',
+  ],
+  [
+    '08/05/2018',
+    '$3,261.10',
+    'Compensation & Pension - Recurring',
+    'Direct Deposit',
+    'BANK OF AMERICA, N.A.',
+    '**3456',
+  ],
+  [
+    '07/05/2018',
+    '$3,261.10',
+    'Compensation & Pension - Recurring',
+    'Direct Deposit',
+    'BANK OF AMERICA, N.A.',
+    '**3456',
+  ],
+  [
+    '06/05/2018',
+    '$3,261.10',
+    'Compensation & Pension - Recurring',
+    'Direct Deposit',
+    'BANK OF AMERICA, N.A.',
+    '**3456',
+  ],
+  [
+    '05/05/2018',
+    '$3,261.10',
+    'Compensation & Pension - Recurring',
+    'Direct Deposit',
+    'BANK OF AMERICA, N.A.',
+    '**3456',
+  ],
+  [
+    '04/05/2018',
+    '$3,261.10',
+    'Compensation & Pension - Recurring',
+    'Direct Deposit',
+    'BANK OF AMERICA, N.A.',
+    '**3456',
+  ],
+  [
+    '03/05/2018',
+    '$3,261.10',
+    'Compensation & Pension - Recurring',
+    'Direct Deposit',
+    'BANK OF AMERICA, N.A.',
+    '**3456',
+  ],
+];
```

Comment:
I chose this demo content because it's the same mock data that was used when the original issue was reported.